signac/__init__.py

```diff
@@ -26,10 +26,10 @@
 )
 from .contrib import filesystems as fs
 from .contrib import get_job, get_project, index, index_files, init_project
+from .contrib.job import get_buffer_load, get_buffer_size
 from .core.h5store import H5Store, H5StoreManager
 from .core.jsondict import JSONDict
```

Comment:
Should this be importing `JSONDict` from synced collections? Would there be API breakage?

Reply:
Yep good catch.